package.json

```diff
@@ -62,6 +63,7 @@
     "react-icons": "^4.2.0",
     "react-moment": "^1.1.1",
     "style-loader": "^3.2.1",
-    "styled-components": "^5.3.1"
+    "styled-components": "^5.3.1",
+    "zlib": "^1.0.5"
```

Comment:
I was under the impression that 'zlib' was included with Node in versions after 10.16. (https://nodejs.org/api/zlib.html)  Do we need to import this?

Reply:
good catch, i uninstalled the zlib node module and our webpack is still working.